winsup/cygwin/dcrt0.cc

```diff
@@ -140,6 +140,8 @@ insert_file (char *name, char *&cmd)
 
   tmp[size++] = ' ';
   strcpy (tmp + size, cmd);
+  if (free_old)
```

Comment:
Can't just do that as cmd does not mean start of the allocated block. Also need to migrate the argv refs if we want to eliminate the strdup (we should). Need some realloc and memmove.